CONTRIBUTING.md

```diff
@@ -1,13 +1,33 @@
+# Getting involved
+
+Since you have landed on this page, we assume that you want to help us with Avy. Amazing. We love it. 
+
+You can start by looking at our list of issues. We do our best to keep the tags on those updated best we can. If you see an issue that you want to tackle, feel free to fork the repo and suggest a PR to fix/add the issues. One of the developers of the project should respond within a few days. If the issue is not assigned to someone, it means no one is currently working on it. 
+
+Things to look at and get engaged on: 
+- [Good first issues](https://github.com/NWACus/avy/labels/good%20first%20issue)
+- [Bugs](https://github.com/NWACus/avy/labels/bug)
+
+If you found an issue you'd like to work on but need more context, it is best to ask questions directly in the issue. The repo issues are monitored by volunteers and NWAC staff who would be able to help provide more details when needed.
+
+You can request to be added to our [volunteer Slack channel](https://join.slack.com/t/nwactechcommittee/shared_invite/zt-30e7qf7xj-7MP7CntTMxTVJ6yusDVsBw) where you can ask questions, sign up to help with larger issues with a more specific timeline and talk with other volunteers who are helping out with Avy. 
+
+Lastly, please fill out the form linked at the [bottom of this page](https://nwac.us/technology-volunteer-outreach/) so we know who you are and what you're interested in!
+
 # Developer Environment
 
-This project uses the Expo framework; follow their excellent [installation guide](https://docs.expo.dev/get-started/installation/) to get a development environment set up. You'll need to [sign up for an account](https://expo.dev/signup) and get invited to the [project](https://expo.dev/accounts/steve.kuznetsov/projects/avalanche-forecast).
+This project uses the Expo framework; follow their excellent [installation guide](https://docs.expo.dev/get-started/installation/) to get a development environment set up. 
+
+If you plan to build the app to run outside of expo go or will need to particiapte in the app publishing process, you'll need to [sign up for an account](https://expo.dev/signup) and get invited to the [project](https://expo.dev/accounts/steve.kuznetsov/projects/avalanche-forecast). If you do not need to do either of those things, you will not need access to the project to proceed with development. 
```

Comment:
```suggestion
If you plan to build a development build of the app using EAS instead of running in the Expo Go sandbox, you'll need to [sign up for an account](https://expo.dev/signup) and get invited to the [project](https://expo.dev/accounts/steve.kuznetsov/projects/avalanche-forecast). If you do not need to do either of those things, you will not need access to the project to proceed with development. 
```

Reply:
We probably want to check - if they don't use EAS (e.g. local machine for build, not cloud), I don't think they even need a log-in. It would just be for folks building e.g. an iOS build without an Apple device and xCode.